src/helpers/vertex.ts

```diff
@@ -14,13 +14,36 @@
  * limitations under the License.
  */
 import { CONFIG } from '../config';
+import { MultiLogger } from './logger';
+
+interface VertexAiPrediction {
+  content: string;
+  safetyAttributes: {
+    blocked: boolean;
+  };
+}
+
+interface VertexAiResponse {
+  predictions: VertexAiPrediction[];
+}
 
 export class VertexHelper {
   private static instance: VertexHelper;
   private projectId: string;
+  private location: string;
+  private endpoint: string;
```

Comment:
We don't need this as a constructor param as mentioned above. it is sufficient to have it under config.vertexAi